cli/Cargo.toml

```diff
@@ -26,8 +26,10 @@ sgx_crypto_helper = { branch = "master", git = "https://github.com/apache/teacla
 # scs / integritee
 substrate-api-client = { features = ["ws-client"], git = "https://github.com/scs/substrate-api-client", branch = "polkadot-v0.9.20-asset" }
 substrate-client-keystore = { git = "https://github.com/scs/substrate-api-client", branch = "polkadot-v0.9.20-asset" }
-my-node-runtime = { git = "https://github.com/integritee-network/integritee-node", branch = "master", package = "integritee-node-runtime" }
-teerex-primitives = { git = "https://github.com/integritee-network/pallets.git", branch = "master" }
+my-node-runtime = { git = "https://github.com/integritee-network/integritee-node", branch = "gk/types_and_utils", package = "integritee-node-runtime" }
+teerex-primitives = { git = "https://github.com/integritee-network/pallets.git", branch = "gk/types_and_utils" }
+itp-types = { git = "https://github.com/integritee-network/pallets.git", branch = "gk/types_and_utils" }
+itp-utils = { git = "https://github.com/integritee-network/pallets.git", branch = "gk/types_and_utils" }
```

Comment:
Although this wasn't the case before, since you're changing this toml file now: Could you sort the entries alphabetically?